_posts/developers/apps/vocabularies/2019-01-01-04_quickstart-artifacts.md

```diff
@@ -22,12 +22,12 @@ const iri = rdf.type;
 ```
 However, an issue with these libraries is that, by their nature, they are limited to just the common, well-known vocabularies. What about your specific vocabulary, designed for your app? How can you make the terms described in that vocabulary easily reusable in your code (and easily reusable in the code of others who may wish to reuse the terms you've defined in your cool vocabulary)?
 
-The LIT artifact generator (which will be made publicly available soon) is a tool that takes any RDF vocabulary as input and automatically generates a nice source-code bundle (i.e. an artifact such as a Java JAR, or a JavaScript NPM module) that you can then easily use in your application. Let's see how it works.
+The LIT artifact generator (publicly available here: https://github.com/inrupt/artifact-generator) is a tool that takes any RDF vocabulary as input and automatically generates a nice source-code bundle (i.e. an artifact such as a Java JAR, or a JavaScript NPM module) that you can then easily use in your application. Let's see how it works.
```

Comment:
Hey @AJamesPhillips  -- thanks for this pr.  It's also available in npm (@inrupt/artifact-generator).  
I can add a suggestion to this or not sure if you prefer to patch it on your own.

For the purpose of this page, the example shown below is probably enough. But, if you were interested in the generator, we do have documentation  https://docs.inrupt.com/developer-tools/javascript/client-libraries/tutorial/generate-vocabulary-artifacts/#generate-vocabulary-artifacts .